src/ontology/mondo-edit.obo

```diff
@@ -617527,6 +617527,18 @@ is_a: MONDO:0019118 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited
 intersection_of: MONDO:0019118 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited retinal dystrophy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9454 {source="https://orcid.org/0000-0002-0587-4693"} ! PROM1
 
+[Term]
+id: MONDO:1040063
+name: KCNV2-related retinopathy
+def: "Any retinopathy caused by variants in the KCNV2 gene." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
```

Comment:
Add also the "mondo gene pattern" as a cross reference (you can look at other examples to find the correct x-ref referring to it)